README.md

```diff
@@ -1,30 +1,129 @@
+8. [Hardware and software architecture](#hardware-and-software-architecture)
+9. [Team members](#team-members)
+
 ## What?
-A software that allows an end-user to remotely control a smart-car in a less traditional way, with the use of an EEG-headset and a mobile app. Moreover, the smart-car can protect itself to prevent for example hitting an obstacle.
+A software that allows an end user to remotely control a smart car in a less traditional way, with the use of an EEG-headset and a mobile app. Moreover, the smart car can prevent itself from hitting obstacles in front and behind it by turning to the opposite direction.
+
 ## Why?
-The main objective of this software is to utilize and demonstrate the use and application of EEG (Electroencephalography) readings to IoT items. By exploring this domain we hope to demonstrate ways using software engineering to develop for people with different levels of mobility.
+The main objective of this software is to utilize and demonstrate the use and application of EEG (Electroencephalography) readings to external hardware. By exploring this domain we hope to demonstrate ways using software engineering to develop for people with different levels of mobility. One of our goals was to showcase the ability to control a car without the use of hands.
+
 ## How?
-The EEG-chip within the “Force Trainer II Bluetooth Headset” measures brain activity in Hz, and depending on users’ mental state e.g. stressed or relaxed, the headset will pick up frequencies within different Hz intervals. With the data retrieved from the headset, we will be able to control the forward throttle of the car, while being able to steer right and left via an app.  
-The smart-car will also be able to prevent itself from colliding with obstacles by using ultrasonic sensors. Once the ultrasonic wave reaches the wall or obstacle it bounces back and the displacement between the car and the wall is calculated, and once it is too short the car will be locked in that direction.
+The EEG-chip within the “Force Trainer II Bluetooth Headset” measures brain, and depending on the users’ mental state e.g. stressed or relaxed, the headset will pick up frequencies within different Hz intervals. With the data retrieved from the headset, we will be able to control the forward throttle of the car, while being able to steer right and left via tilting the phone.
+The smart-car will also be able to prevent itself from colliding with obstacles by using ultrasonic sensors. Once the ultrasonic wave reaches obstacle it bounces back and the displacement between the car and the wall is calculated, and once it is too short the car will turn to the opposite direction.
+
+## System presentation video
+In order to demo our end product, we have recoded a short video demonstrating the functionality of our system. The video can be viewed [here](https://youtu.be/pvArIKRuUSM). We hope the video gives you a good idea of what we've been trying to achieve. Thanks for watching!
```

Comment:
You should "promote" the video more, it's one of the most important ways for others to understand what you did and get inspired.. :)

I'd suggest having its thumbnail with a link instead.

Reply:
Thanks :) I'll have a look at how to do that, didn't know it was possible. Maybe I can link the video thumbnail image on youtube.

Reply:
In your case that would be [this](http://i3.ytimg.com/vi/pvArIKRuUSM/hqdefault.jpg). :+1:

Reply:
I don't know how to re-size images with markdown in the readme, so it'd be good to use an image that is already small, like the youtube thumbnail, so that we don't have to upload a new edit. But that's just cause of lazy.

Reply:
I figured out a way, you can use html tags and set inline styles 👍